tools/gyp/pylib/gyp/common.py

```diff
@@ -470,6 +470,7 @@ def CopyTool(flavor, out_path, generator_flags={}):
         "os400": "flock",
         "solaris": "flock",
         "mac": "mac",
+        "ios": "mac",
```

Comment:
gyp changes should made to https://github.com/nodejs/gyp-next.

Reply:
gyp changes reverted

Reply:
Can you send a PR to the repo ?

Reply:
Send a PR to gyp-next？
How to use gyp-next compile node? Copy  & override correspond dir?

Reply:
https://github.com/nodejs/gyp-next/pull/174